code/game/machinery/Sleeper.dm

```diff
@@ -78,6 +78,10 @@
 	max_chem = E * 20
 	min_health = -E * 25
 
+/obj/machinery/sleeper/Destroy()
+	for(var/mob/M in contents)
+		M.loc = loc

```

Comment:
you need to have a `return ..()` after this or qdel won't know what to do with them